ark/utils/load_utils.py

```diff
@@ -147,6 +152,11 @@ def load_imgs_from_tree(data_dir, img_sub_folder=None, fovs=None, channels=None,
         # get the corresponding indices found in channels_no_delim
         channels_indices = [channels_no_delim.index(chan.split('.')[0]) for chan in all_channels]
 
+        # verify if channels from user input are present in `all_channels`
+        all_channels_no_delim = [channel.split('.')[0] for channel in all_channels]
+        misc_utils.verify_same_elements(channels=all_channels_no_delim,
```

Comment:
These argument names are accurate, but won't help the user understand what they did wrong. When the error message gets raised, we want someone who hasn't looked inside the function to be able to figure out what the problem is